wrappers/python/openmm/app/internal/amber_file_parser.py

```diff
@@ -742,6 +801,34 @@ def readAmberSystem(topology, prmtop_filename=None, prmtop_loader=None, shake=No
         force.addTorsion(iAtom, jAtom, kAtom, lAtom, periodicity, phase, forceConstant)
     system.addForce(force)
 
+    # Add CMAP info.
+    ## Get mapSize and Resolutions
+    numMap = prmtop.getNumMaps()
+    mapSize = prmtop.getCMAPResolutions()
+
+    if numMap and mapSize and numMap == len(mapSize):
```

Comment:
This would become

```python
if numMap > 0 and numMap == len(mapSize):
```

However, does `numMap != len(mapSize)` indicate an error in the file?  If so, we should probably detect that case and throw an exception.

Reply:
At first, I inserted the part for detecting the CMAP parameters in parm7 file, and just tried to add a simple error checking. However, on second thought, there will be no error as far as parm7 files are created by CHARMM-GUI or leaf program. Thus, it would become 
```suggestion
    if numMap > 0:
```

Reply:
No, just push it to your branch and it automatically updates your pull request.